### DISCUSSION TITLE
"EARLY" instead of "MISS" when hitting objects too early

### DISCUSSION THREAD

**Discussion by rvnexe** [ppy/osu#23058] [Ideas]
When you click a hit circle too early, You will get a Miss and a little text appears that says "MISS"
sometimes u may don't notice its diffrence with **too late** misses, since their text is same.
So, it should be better to appear "EARLY" or smth like this instead?

**Comment by ItsShamed** [comment]
Not about misses but still relevant
#3209
#14858
#22603
#22622

**Reply by rvnexe** [reply]
Yeah, but miss is more important, since getting a miss resets your combo.
I'm still wondering how #3209 were closed😂
